documentation/asciidoc/topics/con_cache_cr.adoc

```diff
@@ -9,8 +9,7 @@ When using `Cache` CRs, the following rules apply:
 * `Cache` CRs apply to {datagridservice} pods only.
 * You can create a single cache for each `Cache` CR.
 * If your `Cache` CR contains a template and a cache configuration, {ispn_operator} uses the template.
-* If you edit caches in the {osweb}, the changes are reflected through the user interface but do not take effect on the {brandname} cluster. You cannot edit caches. To change cache configuration, you must first delete the cache through the console or CLI and then re-create the cache.
-* Deleting `Cache` CRs in the {osweb} does not remove caches from {brandname} clusters. You must delete caches through the console or CLI.
+* You cannot edit caches after you create them. To change cache configuration, you must first delete the cache and then re-create it.
```

Comment:
This is not entirely true, you can change the Caches through CR as well as you could through REST api.

Reply:
Thanks, Pavel. I've removed this one.